doc/news/new_opmath.rst

```diff
@@ -161,8 +161,8 @@ Summary of the update
 
     **qml.Hamiltonian**
 
-    The legacy classes :class:`~.pennylane.operation.Tensor` and :class:`~.Hamiltonian` will soon be deprecated.
-    :class:`~.ops.LinearCombination` offers the same API as :class:`~.Hamiltonian` but works well with new opmath classes.
+    The legacy classes :class:`~.pennylane.operation.Tensor` and :class:`~.pennylane.Hamiltonian` will soon be deprecated.
+    :class:`~.ops.op_math.LinearCombination` offers the same API as :class:`~.pennylane.Hamiltonian` but works well with new opmath classes.

```

Comment:
Hamiltonian is not working, but we are already discussing this on Slack

Reply:
Yeah I would consider that a separate issue of no reference to `Hamiltonian` being there in the first place 😬

Reply:
handled in https://github.com/PennyLaneAI/pennylane/pull/5640